src/styles/blocks/arrows.scss

```
@@ -0,0 +1,41 @@
+.stop-watch__arrow {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -100%);
+  animation-timing-function: steps(60, end);
+  animation-iteration-count: infinite;
+  transform-origin: bottom;
+  animation-name: arrows;
+
+  &--sec {
+
+    width: $arrow-width - 3px;
+    height: $arrow-height;
+    border: 1px solid #070707;
+    border-radius: 10px;
+    background-color: #6e0339;
+    animation-duration: 60s;
+
```

redundant empty lines
group modificator`s 
```suggestion
    width: $arrow-width - 3px;
    height: $arrow-height;
    
    border: 1px solid #070707;
    border-radius: 10px;
    background-color: #6e0339;
    
    animation-duration: 60s;
```